### ISSUE TITLE
Error when running ml4f on a Tensorflow.js model with a BatchNormalization layer

### ISSUE THREAD

**Issue by microbit-robert** [microsoft/ml4f#18]
When running ml4f on a Tensorflow.js model with a BatchNormalization layer via the command line, an `Unsupported operator or config` error is thrown. 

Version: 1.9.1
Model: https://github.com/microbit-robert/models/tree/main/1

Please see the full output below.

Input:
`ml4f --no-validate --debug test-model.json`

Output:
```
__________________________________________________________________________________________
Layer (type)                Input Shape               Output shape              Param #
==========================================================================================
input1 (InputLayer)         [[null,24]]               [null,24]                 0
__________________________________________________________________________________________
batch_normalization_BatchNo [[null,24]]               [null,24]                 96
__________________________________________________________________________________________
dense_Dense1 (Dense)        [[null,24]]               [null,16]                 400
__________________________________________________________________________________________
dense_Dense2 (Dense)        [[null,16]]               [null,3]                  51
==========================================================================================
Total params: 547
Trainable params: 499
Non-trainable params: 48
__________________________________________________________________________________________
Layer: InputLayer; data: [24]@0 => [24]@0 (no computation)
Error: Unsupported operator or config: inputShape: 2
    at unsupported (/home/microbit-robert/.local/share/nvm/v20.11.1/lib/node_modules/ml4f/built/cli.cjs:2561:9)
    at Object.compileBatchNorm [as compile] (/home/microbit-robert/.local/share/nvm/v20.11.1/lib/node_modules/ml4f/built/cli.cjs:2941:5)
    at compileModelCore (/home/microbit-robert/.local/share/nvm/v20.11.1/lib/node_modules/ml4f/built/cli.cjs:3305:51)
    at compileModel (/home/microbit-robert/.local/share/nvm/v20.11.1/lib/node_modules/ml4f/built/cli.cjs:4107:16)
    at processModelFile (/home/microbit-robert/.local/share/nvm/v20.11.1/lib/node_modules/ml4f/built/cli.cjs:4740:36)
    at async Object.mainCli (/home/microbit-robert/.local/share/nvm/v20.11.1/lib/node_modules/ml4f/built/cli.cjs:4796:5)
```

Any help with this would be greatly appreciated.

**Comment by mmoskal** [comment]
Are you using latest github version?

**Comment by microbit-robert** [comment]
Yes, it's v1.9.1 downloaded and installed from https://github.com/microsoft/ml4f/releases/tag/v1.9.1.

**Comment by mmoskal** [comment]
so I fixed the batch norm, but see #17

**Comment by mmoskal** [comment]
(you may be able to edit the json to make it work; also even if you change the way you construct the model the weights should stay the same)
